packages/shared/providers.yaml

```diff
@@ -598,16 +701,20 @@ salesforce:
     token_response_metadata:
         - instance_url
     proxy:
-        base_url: ${connectionConfig.instance_url}
+        base_url: https://${connectionConfig.instance_url}.my.salesforce.com/services/data
```

Comment:
This change doesn’t look right. We have active customers using the proxy with sales force with the current setting. What is this change / where did it come from?

Reply:
Since there are existing customers using then I reversed it to the previous state. On the other hand [here](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_list.htm#:~:text=In%20each%20case%2C%20the%20URI%20for%20the%20resource%20follows%20the%20base%20URI%2C%20https%3A//MyDomainName.my.salesforce.com.) is the docs I found that it. Let me know what you think. @khaliqgant

Reply:
I've noticed this style of URL's in Salesforce with some of my customers. It only tends to be for the larger customers though that have something like https://BRAND-NAME.my.salesforce.com